### PR TITLE
Example script

### PR DESCRIPTION
**Pull request by soroushfall**
## Purpose
Add examples to demonstrate usage of the Catalog API Client

## Approach
create separate independent examples that uses API client's methods

## Where should the reviewer start?
examples/

## This PR addresses a:
- [ ] Bug
- [ ] Feature
- [x] Chore

## Quality assurance
locally ran and tested

### CODE REVIEW

**Review by uchenna-cshop — APPROVED** [review]
Good examples! LGTM

File: examples/constants.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+SKU_SOURCE = "us2-test-source"
```
Comment (quentinv72) [inline examples/constants.py:1]: I think this file should be called `data.py` or `mock_data.py`

File: examples/example_01_get_source_uuid.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import os
+import constants
+
+
+"""
+Purpose: Get the source_uuid for a specific source_sku
+"""
+
+
+def get_source_uuid(sku_source):
```
Comment (quentinv72) [inline examples/example_01_get_source_uuid.py:11]: This function shoudl return the `source_uuid` if it exists.
Also you should add type hinting.
`-> Optional[str]`

Comment (quentinv72) [reply examples/example_01_get_source_uuid.py:11]: Also the client should be passed as an argument.

**Review by quentinv72 — CHANGES_REQUESTED** [review]

**Comment by quentinv72** [comment]
It would be better if the methods returned something and didn't just print. That way it'll be closer to how we will use it in our integrations.

File: examples/example_01_get_source_uuid.py (hunk 2 above)
Comment (soroushfall) [reply examples/example_01_get_source_uuid.py]: 👍

File: examples/constants.py (hunk 1 above)
Comment (soroushfall) [reply examples/constants.py]: 👍

**Pull request closed without merging**